opsdroid/core.py

```diff
@@ -74,7 +78,9 @@ def __init__(self, config=None):
         self.stored_path = []
 
     def __enter__(self):
-        """Add self to existing instances."""
+        """Add self to existing instances.
+        
```

Comment:
If you want you could explain what this method does, if you don't wish to do such you can just revert this change back to the previous format.